src/test/java/com/patientpal/backend/caregiver/service/CaregiverServiceTest.java

```diff
@@ -0,0 +1,189 @@
+            assertThat(response.getName()).isEqualTo(caregiver.getName());
+            assertThat(response.getSpecialization()).isEqualTo(caregiver.getSpecialization());
+        }
+
+        @Test
+        void 조회할_때_프로필이_없으면_예외가_발생한다() {
+            // given
+            when(caregiverRepository.findByMember(caregiver.getMember())).thenReturn(Optional.empty());
+
+            // when & then
+            assertThatThrownBy(() -> caregiverService.getProfile(caregiver.getMember().getUsername()))
+                    .isInstanceOf(EntityNotFoundException.class);
+        }
+
+        @Test
+        void 수정을_성공한다() {
+            // given
+            when(caregiverRepository.findByMember(caregiver.getMember())).thenReturn(Optional.of(caregiver));
+            CaregiverProfileUpdateRequest request = updateCaregiverProfileRequest();
+
+            // when
+            caregiverService.updateCaregiverProfile(caregiver.getMember().getUsername(), request);
+
+            // then
+            assertThat(caregiver.getCaregiverSignificant()).isEqualTo(UPDATE_CAREGIVER_SIGNIFICANT);
+            assertThat(caregiver.getExperienceYears()).isEqualTo(UPDATE_EXPERIENCE_YEARS);
+        }
+
+        @Test
+        void 수정할_때_프로필이_없으면_예외가_발생한다() {
+            // given
+            when(caregiverRepository.findByMember(caregiver.getMember())).thenReturn(Optional.empty());
+            CaregiverProfileUpdateRequest request = updateCaregiverProfileRequest();
+
+            // when, then
+            assertThatThrownBy(
+                    () -> caregiverService.updateCaregiverProfile(caregiver.getMember().getUsername(), request))
+                    .isInstanceOf(EntityNotFoundException.class);
+        }
+
+        @Test
+        void 삭제를_성공한다() {
+            // given
+            when(caregiverRepository.findByMember(caregiver.getMember())).thenReturn(Optional.of(caregiver));
+
+            // when
+            caregiverService.deleteCaregiverProfile(caregiver.getMember().getUsername());
+
+            // then
+            verify(caregiverRepository).delete(caregiver);
+            Optional<Caregiver> deletedCaregiver = caregiverRepository.findById(caregiver.getId());
+            assertThat(deletedCaregiver).isEmpty();
+        }
+
+        //TODO 진행중 매칭 있을 시 삭제 불가능 구현 후 수정
+//    @Test
+//    void 삭제할_때_진행_중인_매칭이_있으면_예외가_발생한다() {
+//        // given
+//        when(caregiverRepository.findByMember(caregiver.getMember())).thenReturn(Optional.of(caregiver));
+//        // 진행 중인 매칭 존재 시 예외 발생하도록 설정
+//        doThrow(new IllegalStateException("진행 중인 매칭이 있어 프로필을 삭제할 수 없습니다."))
+//                .when(caregiverRepository).delete(any(Caregiver.class));
+//
+//        // when, then
+//        assertThatThrownBy(() -> caregiverService.deleteCaregiverProfile(caregiver.getMember().getUsername()))
+//                .isInstanceOf(IllegalStateException.class)
+//                .hasMessage("진행 중인 매칭이 있어 프로필을 삭제할 수 없습니다.");
+//    }
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.CommentsIndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Comment has incorrect indentation level 0, expected is 8, indentation should be the same level as line 160.